src/queue/memory.ts

```diff
@@ -0,0 +1,152 @@
+import { EventEmitter } from "stream";
```

Comment:
"stream" is not available on the Browser so we can't use it in adapters that are to be used there.
We can use https://github.com/ai/nanoevents instead, it's already being used in the DocumentDriveServer class